packages/js/src/Novu.ts

```diff
@@ -0,0 +1,39 @@
+import { NovuEventEmitter } from './event-emitter';
+import type { EventHandler, EventNames, Events } from './event-emitter';
+import { Feeds } from './feeds';
+import { Session } from './session';
+import { Preferences } from './preferences';
+
+interface NovuOptions {
+  applicationIdentifier: string;
```

Comment:
This is a good opportunity for a DX improvement. We can encode the `applicationId` in the API key and remove one option.

Reply:
yes, we can, but this requires many other changes... let's leave it for now and improve it later